Dockerfile

```diff
@@ -12,6 +12,42 @@ RUN yum install -y python-requests && \
 RUN mkdir /tftpboot && \
     cp /usr/share/ipxe/undionly.kpxe /usr/share/ipxe/ipxe.efi /tftpboot/
 
+RUN cp /etc/ironic/ironic.conf /etc/ironic/ironic.conf_orig && \
+    crudini --set /etc/ironic/ironic.conf DEFAULT auth_strategy noauth && \
```

Comment:
Appreciate you're just copying existing code around, but wouldn't using `crudini --merge` be more readable?

```
crudini --merge /etc/ironic/ironic.conf <<EOF
[DEFAULT]
auth_strategy=noauth
debug=true

[agent]
deploy_logs_collect=always
EOF
```

Reply:
OH, I've just been wondering if something like that exists, thanks Mark!